js/views/baseVw.js

```diff
@@ -150,6 +150,51 @@ export default class baseVw extends View {
     super.delegateEvents();
   }
 
+  /**
```

Comment:
Useful illustration of current caching versus the new approach. I did see it in there and use it as I began, but the redundancy of having 10 additional methods like the one above led me to the new approach. If we ever do decide to refactor, I could handle the grunt work if you like!